components/local_date_time/local_date_time.tsx

```diff
@@ -48,14 +48,25 @@ class LocalDateTime extends React.PureComponent<Props> {
             titleString = momentDate.toString() + ' (' + momentDate.tz() + ')';
         }
 
-        const timezoneProps = enableTimezone && timeZone ? {timeZone} : {};
         const options = {
-            ...timezoneProps,
-            hour12: !useMilitaryTime,
+            hour: 'numeric',
+            minute: 'numeric',
         };
-        let formattedTime = this.props.intl.formatTime(date, options);
+        if (enableTimezone && timeZone) {
+            options.timeZone = timeZone;
+        }
```

Comment:
looks like this is causing a type check fail - quick fix would be to define `const options: any = {`

Reply:
Thanks, @fm2munsh! I ended up defining an inline interface here. I had hoped to use `Intl.DateTimeFormatOptions`, but found [it does not yet support hourCycle](https://github.com/microsoft/TypeScript/issues/34399).